README.markdown

```diff
@@ -135,13 +135,6 @@ require 'your/app' # Include this line if you want your workers to have access t
 
 ##### Rails 7.1
 
```

Comment:
@dijonkitchen I think this "Rails 7.1" subheading is no longer needed too if you want to send a PR. :bow:

Reply:
Indeed, that makes sense.